### PR TITLE
Update pl.js

### PR DESCRIPTION
**Pull request by kry008**

### CODE REVIEW

File: src/languages/pl.js [hunk 1]
```diff
@@ -195,6 +195,6 @@ export default {
     lunasea: "LunaSea",
     apprise: "Apprise (obsługuje 50+ usług powiadamiania)",
     pushbullet: "Pushbullet",
-    line: "Line Messenger",
-    mattermost: "Mattermost",
+    line: "Czat Line",
+    mattermost: "Mattermost Czat",
```
Comment (Saibamen) [inline src/languages/pl.js:199]: Mattermost is just Mattermost. Why did you add "Chat" suffix?
```suggestion
    mattermost: "Mattermost",
```

**Comment by louislam** [comment]
> I disagree - IMO we shouldn't translate these names

@kry008
